### PR TITLE
tech(component/sidebar): add contextual menu 

### PR DESCRIPTION
**Pull request by sriramanvellingiri**
WIP: Adding the icon in the sidebar
context

### CODE REVIEW

File: react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js [hunk 1]
```diff
@@ -35,6 +38,11 @@ const SidebarNav = (props) => {
   }
 
 
+  const handleClick = (e, data) => {
+    console.log(`Clicked on menu ${data.item}`);
```
Comment (shangan23) [inline react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js:42]: remove console log and unused function

File: react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js [hunk 2]
```diff
@@ -58,6 +67,20 @@ const SidebarNav = (props) => {
...
+
+      <ContextMenu className={styles.menu} id="select_options">
+          <MenuItem onClick={handleClick} data={{item: ""}}
+          className={`${styles.menuItem} ${styles.menuItemFirst}`}>
+              {formatMessage({ id: 'create_sidebar_section' })}
+          </MenuItem>
+          <MenuItem onClick={handleClick} data={{item: ""}} className={styles.menuItem}>
+                {formatMessage({ id: 'browse_channels' })}
+          </MenuItem>
+          <MenuItem onClick={handleChannelAdd} data={{item: "Create a Channel"}} className={styles.menuItem}>
```
Comment (shangan23) [inline react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js:80]: Create a Channel - localise

**Review by shangan23 — APPROVED** [review]
@sriramanvellingiri - left few comments, address in your next pr

File: react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js (hunk 1 above)
Comment (sriramanvellingiri) [reply react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js:42]: Done

File: react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js (hunk 2 above)
Comment (sriramanvellingiri) [reply react/src/components/blocks/Region/Sidebar/SidebarNav/SidebarNav.js:80]: No need of localize and i have updated the code

**Pull request merged**